admin_guide/pruning_resources.adoc

```diff
@@ -188,14 +188,27 @@ particular namespace makes it impossible to calculate their current usage.
 
 |Option |Description
 
+.^|`--all`
+|Include images that were not pushed to the registry but have been mirrored by
+pullthrough. This is on by default. To limit the pruning to images that were
+pushed to the integrated registry, pass `--all=false`.
+
 .^|`--certificate-authority`
 |The path to a certificate authority file to use when communicating with the
 {product-title}-managed registries. Defaults to the certificate authority data
-from the current user's config file.
+from the current user's config file. If provided, secure connection will be
+initiated.
 
 .^|`--confirm`
 |Indicate that pruning should occur, instead of performing a dry-run.
 
+.^|`--force-insecure`
+|Allow an insecure connection to the docker registry that is hosted
+via HTTP or has an invalid HTTPS certificate. By default, insecure connection
+is allowed only in case where neither certificate-authority nor registry-url is
+specified. Whenever possible, use `--certificate-authority` instead of this
```

Comment:
I've ditched it. There is plenty of other places advocating the same. Instead, I've put *(Dangerous)* at the beginning.